csuntechlab/iSTART
# STR-106 -- FE: Create Actual Control group page

# Description
'Write a meaningful description of your task'
  1. Create Control Group's dashboard, being a message portraying that they're done with the research
# Testing
'What does the reviewer need to do to test your PR?'
  1. Login as someone in the control group, being `alice.lu`
  2. When you login you should be directed to the dashboard for control group, which is just a message
  3.Test the responsiveness
  4. Make sure it is like the mockup
  5. Look at my SASS & BEM
